tests/tests_pytorch/trainer/connectors/test_accelerator_connector.py

```diff
@@ -484,7 +484,6 @@ def test_strategy_choice_ddp_spawn(cuda_available_mock, device_count_mock):
 @pytest.mark.parametrize("strategy", ["ddp", DDPStrategy()])
 def test_strategy_choice_ddp_slurm(setup_distributed_mock, strategy):
     trainer = Trainer(fast_dev_run=True, strategy=strategy, accelerator="gpu", devices=2)
-    assert trainer._accelerator_connector._is_slurm_managing_tasks()
```

Comment:
Any reason why you didnt add `assert SLURMEnvironment.detect()`

Reply:
I updated the test to cover both cases